src/main/java/com/squareup/kotlinpoet/FunSpec.kt

```diff
@@ -342,6 +359,8 @@ class FunSpec private constructor(builder: Builder) {
     internal const val CONSTRUCTOR = "constructor()"
     internal const val GETTER = "get()"
     internal const val SETTER = "set()"
+    internal const val RETURN = "return "
+    @JvmField internal val RETURN_CODE_BLOCK = CodeBlock.of(RETURN)
```

Comment:
For Android apps and libraries such a micro-optimization is probably useful, but for libraries we want to be as generic and idiomatic as possible. Plus we want to reduce our dependence on JVM-specific bits to hopefully allow this library to work on node.js or via native code someday.